alerts/kubelet.libsonnet

```
@@ -101,6 +122,22 @@ local utils = import '../lib/utils.libsonnet';
               summary: 'Node readiness status is flapping.',
             },
           },
+          {
+            alert: 'KubeNodeEvictions',
+            expr: |||
+              sum(rate(kubelet_evictions{%(kubeletSelector)s}[15m])) by(%(clusterLabel)s, eviction_signal, node) > %(evictionRateThreshold)s
+            ||| % $._config,
+            labels: {
+              severity: 'info',
+            },
+            'for': '0s',
+            annotations: {
+              description: 'Node {{ $labels.node }}%s is evicting Pods due to {{ $labels.eviction_signal }}.  Eviction occurs when eviction thresholds are crossed, typically caused by Pods exceeding RAM/ephemeral-storage limits.' % [
+                utils.ifShowMultiCluster($._config, ' in {{ $labels.%(clusterLabel)s }}' % $._config),
```

nit, only to be consistent with all the other uses

```suggestion
                utils.ifShowMultiCluster($._config, ' on {{ $labels.%(clusterLabel)s }}' % $._config),
```